django/contrib/admin/actions.py

```diff
@@ -33,8 +33,11 @@ def delete_selected(modeladmin, request, queryset):
 
     # Populate deletable_objects, a data structure of all related objects that
     # will also be deleted.
-    deletable_objects, model_count, perms_needed, protected = get_deleted_objects(
-        queryset, opts, request.user, modeladmin.admin_site, using)
+    deletable_objects, model_count, perms_needed, protected = (
```

Comment:
I'm not sure if this could be classified as a backwards-incompatible change. Essentially, if you override the `get_deleted_objects()` method on the `ModelAdmin` you suddenly have a different behavior when you select the "Delete Selected" action. It might delete more/less than it used to.

Not that this is a big deal, but it might be worth noting in the release notes. Thoughts @felixxm?